README.md

```diff
@@ -1,2 +1,6 @@
 # Slothstagram
 
+# Setting up Django Backend, Installing Dependencies
+- ```git clone``` the repo
+- View ```requirements.txt``` for dependencies
+- Run ```pip install django django-admin``` to install dependencies
```

Comment:
1. Include the steps for creating a virtual environment with [virtualenvwrapper](https://virtualenvwrapper.readthedocs.io/en/latest/index.html).
2. The command to install dependencies will be `pip install -r requirements.txt` rather than having to specify dependencies manually